flink-runtime/src/test/java/org/apache/flink/runtime/jobmaster/slotpool/SlotPoolTest.java

```diff
@@ -97,34 +94,30 @@
 
 	private TestingResourceManagerGateway resourceManagerGateway;
 
+	private ComponentMainThreadExecutor mainThreadExecutor =
+		TestingComponentMainThreadExecutorServiceAdapter.forMainThread();
```

Comment:
Do we need concurrency in this test case?

Reply:
It was already not concurrent anymore, so yes we can simplify.